rest/utilities_testing.go

```diff
@@ -46,8 +46,27 @@ type RestTester struct {
 	InitSyncSeq             uint64 // If specified, initializes _sync:seq on bucket creation.  Not supported when running against walrus
 }
 
+type RestTester struct {
+	*RestTesterConfig
+	Testing testing.TB
+}
+
+func NewRestTester(Tester testing.TB, RestConfig *RestTesterConfig) *RestTester {
```

Comment:
Parameters in functions usually start with lowercase by convention.

```suggestion
func NewRestTester(tester testing.TB, restConfig *RestTesterConfig) *RestTester {
```